addons/captives/XEH_postInit.sqf

```diff
@@ -41,7 +41,7 @@ if (!hasInterface) exitWith {};
     private _target = cursorObject;
     if !([ACE_player, _target, []] call EFUNC(common,canInteractWith)) exitWith {false};
     if !(_target isKindOf "CAManBase") exitWith {false};
-    if ((_target distance ACE_player) > getNumber (configFile >> "CfgVehicles" >> "CAManBase" >> "ACE_Actions" >> "ACE_ApplyHandcuffs" >> "distance")) exitWith {false};
+    if ((_target distance ACE_player) > HANDCUFFS_DISTANCE) exitWith {false};
```

Comment:
Leaving it as a config lookup would allow other mods to adjust the distance if wanted.
```suggestion
    if ((_target distance ACE_player) > getNumber (configOf ACE_player >> "ACE_Actions" >> "ACE_ApplyHandcuffs" >> "distance")) exitWith {false};
```